thaw_components/src/binder/mod.rs

```diff
@@ -182,7 +184,8 @@ fn FollowerContainer<El: ElementDescriptor + Clone + 'static>(
         let mut style = String::new();
         if let Some(width) = width {
             let width = match width {
-                FollowerWidth::Target => format!("width: {}px;", target_rect.width()),
+                FollowerWidth::Target => format!("min-width: {}px;", target_rect.width()),
+                FollowerWidth::MinTarget => format!("width: {}px;", target_rect.width()),
```

Comment:
Oops, `MinTarget` should've been `min-width` instead of `Target`.
I added `MinTarget` and used it in the `Select` now to make sure the select dropdow menu doesn't get cut off if the select input is too short. I think this would be fine to have the dropdown menu be longer in case the options are longer, but its up to you to keep this or not. Personally as a user of Thaw, I'd prefer the text in the dropdown not to be cut off